File: store/src/main/java/com/nytimes/android/external/store2/base/impl/StoreDefaults.java
package com.nytimes.android.external.store2.base.impl;

import java.util.concurrent.TimeUnit;

final class StoreDefaults {
    private StoreDefaults() {

    }

    /**
     * Default Cache TTL, can be overridden
     *
     * @return memory persister ttl
     */
    static long getCacheTTL() {
        return TimeUnit.HOURS.toSeconds(24);
    }

    /**
     * Default mem persister is 1, can be overridden otherwise
     *
     * @return memory persister size
     */
    static long getCacheSize() {
        return 100;
    }

    static TimeUnit getCacheTTLTimeUnit() {
        return TimeUnit.SECONDS;
    }
}

File: store/src/main/java/com/nytimes/android/external/store3/base/impl/StoreDefaults.kt
package com.nytimes.android.external.store3.base.impl

import java.util.concurrent.TimeUnit

internal object StoreDefaults {

    /**
     * Default Cache TTL, can be overridden
     *
     * @return memory persister ttl
     */
    val cacheTTL: Long
        get() = TimeUnit.HOURS.toSeconds(24)

    /**
     * Default mem persister is 1, can be overridden otherwise
     *
     * @return memory persister size
     */
    val cacheSize: Long
        get() = 100

    val cacheTTLTimeUnit: TimeUnit
        get() = TimeUnit.SECONDS
}
